packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.9.0",
+  "version": "6.16.1",
```

Comment:
Caused by dependencies update

Reply:
Good point, I think it makes sense to skip given that upgraded dependencies are not pinned to specific versions (they use range version), so the users can upgrade `@mui/x-license-pro`'s dependencies even if we don't release the package.

Reply:
Yes, that's my point. 💡 
If we used direct dependency versions, then sure, we would basically be forced to release every time. 🤔 👌